test/unit/service.spec.js

```diff
@@ -1442,10 +1442,21 @@ describe("Test Service", () => {
 			expect(
 				contextFn({
 					connection: {
-						$service: "service",
+						context: {
+							$service: "service",
+							socket: {
+								$ctx: "context",
```

Comment:
Saw that the CI failed here.  The `$ctx` and `$params` need to be moved up a level now.